### PR TITLE
Update adax.markdown

### PR DESCRIPTION
**Pull request by mdehaas**
## Proposed change
Seems the app has changed slightly since this documentation was written. This text reflects the current app version.

## Type of change
Changes to creating integration account

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by Danielhiversen — DISMISSED** [review]
Thanks

File: source/_integrations/adax.markdown [hunk 1]
```diff
@@ -22,9 +22,10 @@ The local integrations only works with newer Adax heaters with both Bluetooth an
 
 For the cloud integration, you'll need the Account ID (which can be found in the Adax Wifi app, pressing 'Account'). You will also need a credential, which you can create in the Adax app:
 
-1. Navigate to Account Tab,
-2. Select “Remote user client API”
-3. Select “Add Credential”
-4. Give some name to the created credential and copy the generated password.
+1. Navigate to Account Tab
```
Comment (c0ffeeca7) [inline source/_integrations/adax.markdown]:
```suggestion
1. Navigate to the Account tab.
```

File: source/_integrations/adax.markdown [hunk 2]
```diff
@@ -22,9 +22,10 @@ The local integrations only works with newer Adax heaters with both Bluetooth an
...
+2. Go to "Third party integrations"
```
Comment (c0ffeeca7) [inline source/_integrations/adax.markdown]:
```suggestion
2. Go to **Third party integrations**.
```

File: source/_integrations/adax.markdown [hunk 3]
```diff
@@ -22,9 +22,10 @@ The local integrations only works with newer Adax heaters with both Bluetooth an
...
+3. Select “Remote API”
```
Comment (c0ffeeca7) [inline source/_integrations/adax.markdown]:
```suggestion
3. Select **Remote API**.
```

File: source/_integrations/adax.markdown [hunk 4]
```diff
@@ -22,9 +22,10 @@ The local integrations only works with newer Adax heaters with both Bluetooth an
...
+4. Select “Add Credential”
```
Comment (c0ffeeca7) [inline source/_integrations/adax.markdown]:
```suggestion
4. Select **Add Credential**.
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @mdehaas :+1:

**Pull request merged**